salt/fileclient.py

```diff
@@ -980,15 +983,26 @@ def get_file(self,
                 else:
                     return False
             fn_ = salt.utils.fopen(dest, 'wb+')
-        while True:
+        # prevent endless loops
+        tries = 1000000
+
+        while tries > 0:
+            tries = tries  -1
+            if tries == 0:
```

Comment:
this if block is dead code and cannot be reached, please remove it, the while should be =>